cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -260,3 +288,46 @@ def test_custom_button_automate(request, appliance, context, submit, objects, bu
                 assert False, "Expected {count} requests not found in automation log".format(
                     count=str(expected_count)
                 )
+
+
+@pytest.mark.meta(
+    blockers=[
+        BZ(
+            1659452,
+            forced_streams=["5.9", "5.10"],
+            unblock=lambda serv_button_group: "group" not in serv_button_group,
+        )
+    ]
+)
+@pytest.mark.parametrize("context", [ViaUI, ViaSSUI])
+def test_custom_button_text_display(appliance, context, serv_button_group, service):
+    """ Test custom button text display on option
+
+    Bugzilla:
+        * 1650066
+
+    Polarion:
+        assignee: ndhandre
+        caseimportance: medium
+        initialEstimate: 1/6h
+        testSteps:
+                1. Appliance with Service
+                2. Create custom button `Group` or `Button` without display option
+                3. Check Group/Button text display or not on UI and SSUI.
+    """
+
+    my_service = MyService(appliance, name=service.name)
+    button, group = serv_button_group
+
+    with appliance.context.use(context):
+        navigate_to = ssui_nav if context is ViaSSUI else ui_nav
+        destinations = (
+            ["Details"] if (BZ(1650066, forced_streams=["5.9", "5.10"]).blocks and context is ViaSSUI) else ["All", "Details"]
```

Comment:
Could you double check the formatting here? I think it is right when looking at the raw text.

Reply:
@jawatts those are right.

Reply:
This appears to be 127 char long?